### PR TITLE
EJ2-Documentation(849146)- UG Documentation for ColumnChooser

### PR DESCRIPTION
**Pull request by Gayathri4135**
UG Documentation for ColumnChooser - Added a new topic hide columns in column chooser dialog, customize column chooser dialog size, change default search operator of the column  chooser, diacritics searching  in column chooser.

### CODE REVIEW

File: ej2-javascript/code-snippet/grid/columnchooser-changeoperator-cs1/index.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Grid.Inject( Toolbar, ColumnChooser);
+
+let grid: Grid = new Grid({
+    dataSource: data,
+    showColumnChooser: true,
+    toolbar: ['ColumnChooser'],
+    columnChooserSettings: { operator: 'contains' },
+    columns: [
+        { field: 'OrderID', headerText: 'Order ID', width: 120, textAlign: 'Right', showInColumnChooser: false },
```
Comment (BalajiSGrid) [inline ej2-javascript/code-snippet/grid/columnchooser-changeoperator-cs1/index.ts:12]: is showInColumnChooser need on this?

File: ej2-javascript/grid/columns/column-chooser.md [hunk 2]
```diff
@@ -9,9 +9,9 @@ documentation: ug
 domainurl: ##DomainURL##
 ---
 
-# Column chooser in ##Platform_Name## Grid control
+# Column chooser in ##Platform_Name## Grid component
```
Comment (BalajiSGrid) [inline ej2-javascript/grid/columns/column-chooser.md:12]: use control instead component

**Comment by BalajiSGrid** [comment]
Need check demo

**Comment by Gayathri4135** [comment]
rebuild

File: ej2-javascript/code-snippet/grid/columnchooser-changeoperator-cs1/index.ts (hunk 1 above)
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/columnchooser-changeoperator-cs1/index.ts]: No, needed , Remove the showInColumnChooser in orderID

File: ej2-javascript/grid/columns/column-chooser.md (hunk 2 above)
Comment (Gayathri4135) [reply ej2-javascript/grid/columns/column-chooser.md]: change "component" to "control"

File: ej2-javascript/grid/columns/column-chooser.md [hunk 3]
```diff
@@ -9,9 +9,9 @@ documentation: ug
...
 
-The column chooser has options to show or hide columns dynamically. It can be enabled by defining the [`showColumnChooser`](../../api/grid/#showcolumnchooser) as true.
+The column chooser feature in the Syncfusion ##Platform_Name## Grid component allows you to dynamically show or hide columns. This feature can be enabled by defining the [showColumnChooser](../../api/grid/#showcolumnchooser) property as **true**.
```
[the remark this answers is not in the document]
Comment (Gayathri4135) [reply ej2-javascript/grid/columns/column-chooser.md]: change "component" to "control"

**Comment by BalajiSGrid** [comment]
Technical review and demo has been completed.

**Review by MaithiliyK — APPROVED** [review]
changes are fine

**Pull request merged**